src/racing/actors_extended.c

```diff
@@ -547,7 +549,7 @@ s32 func_802B18E4(Player *player, s16 tripleShellType) {
 }
 
 // This function could reasonably be called "spawn_shell_for_triple_shell" or similar
```

Comment:
remove comment.

Should it be named spawn or use instead of init?

Reply:
for me spawn it's when it only add in actorlist here he add but also he give some property to the shell object

Reply:
Spawn should generally be considered "spawned into the map"

Lets take garrys mod for instance. If I add a vehicle to the game I spawned that vehicle.

It does make sense that if you have a lightning bolt item, that's it's been "used" not spawned. But even for a shell, at some point it's been spawned into the map. But use/spawn here is close enough it's good enough.

Reply:
ok